parcels/examples/example_nemo_curvilinear.py

```diff
@@ -29,7 +29,7 @@ def run_nemo_curvilinear(mode, outfile, advtype='RK4'):
                        'data': data_path + 'V_purely_zonal-ORCA025_grid_V.nc4'}}
     variables = {'U': 'U', 'V': 'V'}
     dimensions = {'lon': 'glamf', 'lat': 'gphif'}
-    field_chunksize = {'y': 2, 'x': 2}
+    field_chunksize = {'y': 256, 'x': 512}
```

Comment:
Because otherwise running the example would become _extremely_ slow, timing out the 6-hour max of github CI. 

Note that previously these fields were not actually loaded with dask, as they have less than three time snapshots. So this would be loaded in netcdf, and the `field_chunksize` was irrelevant. 

In the new implementation, we also have a non-deferred dask fieldfilebuffer (which could be useful for e.g. super-fine-resolution 0.01 degree global bathymetry Fields), so the `field_chunksize` is now used. And that blew out the CI ;-)

Makes sense?

Reply:
Makes total sense - yes, with chunks of 2x2, if the field is more than 100x100 cells, managing the individual chunks becomes the main bottleneck in Dask. Absolutely understandable reason.